doc/api/fs.md

```diff
@@ -2749,8 +2749,27 @@ changes:
 * `callback` {Function}
   * `err` {Error}
 
-Asynchronous unlink(2). No arguments other than a possible exception are given
-to the completion callback.
+Asynchronously removes a file or symbolic link. No arguments other than a
+possible exception are given to the completion callback.
+
+```js
+// Assuming that 'path/file.txt' is a regular file.
+fs.unlink('path/file.txt', (err) => {
+  if (err) throw err;
+  console.log('path/file.txt was deleted');
+});
+```
+
+`fs.unlink()` will not work on a directory, empty or otherwise. To remove a directory, use `fs.rmdir()`.
```

Comment:
It would be good to actually make this a link. Most functions are already listed but `rmdir` does not seem to be one of those. So please add the entry at the bottom of the file and write here: 

```
[`fs.rmdir()`][]
```